apps/api/src/python/query/index.ts

```diff
@@ -409,16 +417,25 @@ def _briefer_list_dataframes():
                             categories = df[col["name"]].dropna().unique()
                             categories = list(categories)
                             categories = list(dict.fromkeys(categories))
-                            categories = categories[:1000]
+                            categories = categories[:1000] 
                             col["categories"] = categories
                         except:
                             pass
+                    # Handle datetime and date types by converting them to strings
+                    elif pd.api.types.is_datetime64_any_dtype(dtype):
+                        df[col["name"]] = df[col["name"]].apply(
+                            lambda x: x.isoformat() if isinstance(x, (datetime, date)) else x
+                        )  # Only apply isoformat to datetime objects
 
                 dataframes.append({"name": name, "columns": columns})
-        except:
+        except Exception as e:
+            print(f"Error processing dataframe '{name}': {e}")
```

Comment:
This print is never seen anywhere, we should just remove. This `except` is here just so that we do not fail the whole thing if for some reason we fail to compute columns of one of the dataframes.
```suggestion
```